Examples/Swift/ExtrusionsExample.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ExtrusionsExample.swift
+//  Examples
+//
+//  Created by Jordan Kiley on 5/10/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+import Mapbox
+
+@objc(ExtrusionsExample_Swift)
+
+class ExtrusionsExample: UIViewController, MGLMapViewDelegate {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let mapView = MGLMapView(frame: view.bounds, styleURL: MGLStyle.lightStyleURL(withVersion: 9))
+        
+        // Center the map on the Colosseum in Rome, Italy.
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 41.8902, longitude: 12.4922), animated: false)
+
+        // Set the map view camera's pitch and distance.
+        mapView.camera = MGLMapCamera(lookingAtCenter: mapView.centerCoordinate, fromDistance: 600, pitch: 60, heading: 0)
+        mapView.delegate = self
+        
+        view.addSubview(mapView)
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        
+        // Access the Mapbox Streets source and use it to create a `MGLFillExtrusionLayer`.
+        let source = style.source(withIdentifier: "composite")
+        let layer = MGLFillExtrusionStyleLayer(identifier: "buildings", source: source!)
+        layer.sourceLayerIdentifier = "building"
+        
+        // Filter out buildings that should not extrude.
+        layer.predicate = NSPredicate(format: "extrude != false AND height >=0")
```

Comment:
From what I've read [here](https://www.mapbox.com/vector-tiles/mapbox-streets-v7/), `extrude` may be `false` if it is a complex building that requires a lot of building parts to render it correctly (like the Colosseum). This tells the renderer to skip rendering the overall building, and instead render the individual building parts. 

This is mostly to catch some strange rendering that @incanus and I noticed when we were testing this out on Lower Manhattan. (which is a bug that he reported)

<img width="301" alt="screen shot 2017-05-10 at 11 41 44 am" src="https://cloud.githubusercontent.com/assets/12474734/26017591/32cc031e-371f-11e7-91b2-e5b297a9e4d7.png">